frontend/src/styles.scss

```
@@ -101,3 +101,142 @@ body {
+        background: rgba(0, 0, 0, 0.2);
+        z-index: 10;
+    }
+}
+
+@media (prefers-color-scheme: dark) and (width <= 600px) {
+    .mobile-bottom-sheet-dialog .mat-mdc-dialog-surface::before {
+        background: rgba(255, 255, 255, 0.24);
+    }
+}
+
+@keyframes mobileBottomSheetSlideUp {
```

Stylelint (17.12.0)

[error] Expected keyframe name "mobileBottomSheetSlideUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)